mqtt/mqtt.c

```diff
@@ -438,8 +469,9 @@ mqtt_send_message(MQTT_Client* client) {
   // get some details about the message
   uint16_t msg_type = mqtt_get_type(buf->data);
   uint8_t  msg_id = mqtt_get_id(buf->data, buf->filled);
+  msg_id = msg_id;
```

Comment:
Was a workaround for unused variable before I added the ignore to the
makefile
On Sep 12, 2015 12:18 AM, "Thorsten von Eicken" <notifications@github.com>
wrote:

> In mqtt/mqtt.c
> <https://github.com/jeelabs/esp-link/pull/47#discussion_r39334354>:
>
> > @@ -438,8 +469,9 @@ mqtt_send_message(MQTT_Client* client) {
> >    // get some details about the message
> >    uint16_t msg_type = mqtt_get_type(buf->data);
> >    uint8_t  msg_id = mqtt_get_id(buf->data, buf->filled);
> > +  msg_id = msg_id;
>
> ??
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jeelabs/esp-link/pull/47/files#r39334354>.
>